<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWdialogLocate</class>
 <widget class="QDialog" name="QWdialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>单元格设定与文字</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>列号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBoxColumn"/>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="checkBoxColumn">
      <property name="text">
       <string>列增</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>行号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBoxRow"/>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBoxRow">
      <property name="text">
       <string>行增</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>设定文字</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="edtCaption"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnSetText">
      <property name="text">
       <string>设定文字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWdialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
